prompt_toolkit/formatted_text/pygments.py

```diff
@@ -19,6 +19,9 @@ def __pt_formatted_text__(self):
         result = []
 
         for token, text in self.token_list:
-            result.append(('class:' + pygments_token_to_classname(token), text))
+            if list(token) == ['ZeroWidthEscape']:
```

Comment:
Why is this needed? A Pygments lexer should never yield "ZeroWidthEscape".
The idea of this class is only to provide compatibility with whatever Pygments generates.